### PR TITLE
feat: add EU Consolidated Sanctions List

### PR DESCRIPTION
**Pull request by diginicho**
This PR is handles issue: 184 - adding EU Consolidated Sanctions List data and makes it available to the searching paths in both the main search and its own endpoint similar to the US data.

### CODE REVIEW

File: pkg/csl/download_eu.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+package csl
+
+import (
+	"fmt"
+
+	"github.com/moov-io/base/log"
+	"github.com/moov-io/watchman/pkg/download"
+)
+
+const uri = "https://webgate.ec.europa.eu/fsd/fsf/public/files/csvFullSanctionsList_1_1/content?token=dG9rZW4tMjAxNw"
```
Comment (adamdecaf) [inline pkg/csl/download_eu.go:14]: What's this `token` from? I assume we can't use the same token for every user of Watchman.

Comment (diginicho) [reply pkg/csl/download_eu.go:14]: That uri is pulled from the direct access link from the European Union website: https://data.europa.eu/data/datasets/consolidated-list-of-persons-groups-and-entities-subject-to-eu-financial-sanctions?locale=en

Comment (diginicho) [reply pkg/csl/download_eu.go:14]: Screen Shot 2022-11-15 at 9 30 29 AM

Comment (adamdecaf) [reply pkg/csl/download_eu.go:14]: `strx.Or(..)` comes from moov-io/base's `strx` package.

Comment (diginicho) [reply pkg/csl/download_eu.go:14]: for sure! I also reached out on their contact us page to see if that changes and how often

File: cmd/server/search_handlers.go [hunk 2]
```diff
@@ -321,10 +326,23 @@ var (
...
+			resp.EUCSL = s.TopEUCSL(limit, minMatch, name)
+		},
+	}
 )
 
+func generateAllGatherings() []searchGather {
+	gatherings = append(gatherings, cslGatherings...)
+	gatherings = append(gatherings, euGatherings...)
+	return gatherings
+}
```
Comment (adamdecaf) [inline cmd/server/search_handlers.go:342]: I don't think we want to append these to global state? With each request we'd grow the array of searchGathers over and over.

**Review by adamdecaf — APPROVED** [review]

File: cmd/server/search_handlers.go (hunk 2 above)
Comment (diginicho) [reply cmd/server/search_handlers.go]: good catch, I updated this to be a var block like before

**Review by adamdecaf — APPROVED** [review]

**Pull request merged**